app/screens/search/search.js

```diff
@@ -116,10 +123,12 @@ export default class Search extends PureComponent {
         if (shouldScroll) {
             setTimeout(() => {
                 const modifiersCount = enableDateSuggestion ? 5 : 2;
-                this.refs.list._wrapperListRef.getListRef().scrollToOffset({ //eslint-disable-line no-underscore-dangle
-                    animated: true,
-                    offset: SECTION_HEIGHT + (modifiersCount * MODIFIER_LABEL_HEIGHT) + (recentLength * RECENT_LABEL_HEIGHT) + ((recentLength + 1) * RECENT_SEPARATOR_HEIGHT),
-                });
+                if (this.refs.list) {
+                    this.refs.list._wrapperListRef.getListRef().scrollToOffset({ //eslint-disable-line no-underscore-dangle
```

Comment:
why would the list not be ready here, that you need to check for the existence of the ref? and if that is the case when will you trigger the needed scrollToOffset ?

Reply:
I forget exactly how I hit an error here, but since this is called in a `setTimeout`, the component may have been unmounted by this point. Ideally, we just wouldn't use a `setTimeout` here, but I haven't looked into why it was added in the first place